server/mailet/mailets/src/test/java/org/apache/james/transport/mailets/delivery/SieveIntegrationTest.java

```diff
@@ -937,6 +938,15 @@ public void vacationDaysAddressesShouldWork() throws Exception {
             .build();
         assertThat(fakeMailContext.getSentMails()).containsExactly(expectedSentMail);
     }
+    
+    @Test(expected = MessagingException.class)
```

Comment:
No we don't expect a MessagingException.

We want the Sieve mailet to perform normally IE do nothing, upon sieve notification, isn't it? 

```
assertThat(mail.getAttribute(ATTRIBUTE_NAME)).isEmpty()
```

Should be enough to ensure Sieve was not executed...